ios/Sources/GutenbergKit/Sources/EditorConfiguration.swift

```diff
@@ -26,16 +26,10 @@ public struct EditorConfiguration: Sendable {
     public let namespaceExcludedPaths: [String]
     /// Authorization header
     public let authHeader: String
-    /// Raw block editor settings from the WordPress REST API
-    public let editorSettings: String
     /// Locale used for translations
     public let locale: String
     /// Enables the native inserter UI in the editor
     public let isNativeInserterEnabled: Bool
-    /// Endpoint for loading editor assets, used when enabling `shouldUsePlugins`
-    public var editorAssetsEndpoint: URL?
-    /// Logs emitted at or above this level will be printed to the debug console
-    public let logLevel: LogLevel
```

Comment:
Are host apps still able to configure the log levels emitted from the JavaScript logger utility?